src/renderer/shared/ui/Markdown/Markdown.tsx

```diff
@@ -72,17 +73,16 @@ const components: Components = {
       <Checkbox
         {...props}
         onChange={(newCheckedState: boolean) => {
-          if (typeof props.onChange === 'function') {
```

Comment:
Why do you need this change?

Reply:
fast `return` leads to less nesting

Reply:
Is it necessary in context of this PR?

Reply:
Just a little 😉
For me it's quite natural to improve readability with minor rearrangements 
If you're strongly against this practice, I will hold myself